test/guest/cockpit-fedora-22.setup

```diff
@@ -68,6 +68,10 @@ dnf -y upgrade
 #
 dnf -y copr enable phatina/storaged
 
-dnf -y install $TEST_PACKAGES $COCKPIT_DEPS $IPA_CLIENT_PACKAGES
+# dnf in Fedora 22 has a bug where it'll ignore packages that
+# it fails to find (for whatever reason)
+for dep in $TEST_PACKAGES $COCKPIT_DEPS $IPA_CLIENT_PACKAGES: do
```

Comment:
I don't think this is true.  `yum` behaves like that, but not `dnf` as far as I can tell.  This exits non-zero, for example:

```
# dnf install dnf blurg
Last metadata expiration check performed 1:59:19 ago on Fri Oct 23 16:43:26 2015.
Package dnf-1.1.2-4.fc22.noarch is already installed, skipping.
No package blurg available.
Error: Unable to find a match.
# echo $?
1
```

However, we didn't run the setup script with -e until recently.

Also, I think it is better to give the complete problem to `dnf` so that it might find a better solution.  Indeed, if you call it twice, installing the second package might uninstall the first package, without any error indication.

If we can not trust the exit code, I would rather check afterwards with rpmquery whether all packages are acually installed.